context/app/static/js/shared-styles/inputs/MultiAutocomplete/MultiAutocomplete.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+
+import Autocomplete, { AutocompleteProps } from '@mui/material/Autocomplete';
+import TextField, { TextFieldProps } from '@mui/material/TextField';
+import { InputLabelProps as inputLabelProps } from '@mui/material/InputLabel';
+
+interface MultiAutocompleteTypes extends AutocompleteProps {
+  tagComponent: React.ReactNode;
```

Comment:
`tagComponent` should be typed as `ComponentType`  instead of `ReactNode`, since `ComponentType` refers to the actual component name that would be passed around while `ReactNode` is the result of a component being rendered.

e.g.:
```
function Foo() {
    return <div>bar</div>
}
```
In this example, `Foo` is a `ComponentType` while `<div>bar</div>` is a `ReactNode`